### ISSUE TITLE
Program shows order placed,but in actual order wasn't placed

### ISSUE THREAD

**Issue by Bhavnak13** [pcomputo/Whole-Foods-Delivery-Slot#43]
Thank yo for sharing code,

In my console I do see print statements as below but when checked in amazon site my order wasn't placed.
What could be the reason, any clues?
console statements for whole_foods_delivery_autobuy_windows.py(using the latest code of 11th Apr)
Selected slot and continued to next page
Payment method selected
Order reviewed
Order Placed!

**Comment by pcomputo** [comment]
Interesting. After the program finished did you check the screen it was running on? What was the page it stopped at? Ideally should've been the order placed page.

**Comment by Bhavnak13** [comment]
The chrome window got closed and my command prompt had the message I shared before.
In fact similar thing happened again but this time there was error after order placed
Selected slot and continued to next page
Payment method selected
Order reviewed
Order Placed!
[55132:47684:0412/150039.540:ERROR:broker_win.cc(55)] Error reading broker pipe: The pipe has been ended. (0x6D)
[55132:47684:0412/150039.540:ERROR:broker_win.cc(141)] Error sending sync broker message: The pipe is being closed. (0xE8).

Since 6 days I am running this program, but not able to get the slot but I hope this program is the only hope for me else manually refreshing and grabbing a slot is impossible.

Thanks for sharing and giving your inputs.
